BGAnimations/ScreenGameplay underlay/PerPlayer/Score.lua

```diff
@@ -27,9 +26,9 @@ return LoadFont("_wendy monospace numbers")..{
 	Name=pn.."Score",
 	InitCommand=function(self)
 		self:valign(1):halign(1)
-                self:zoom(Positions.ScreenGameplay.ScoreZoom())
```

Comment:
Wouldn't it make sense to use Positions.whatever here?

Reply:
Nah — it was an attempt to standardize where things are on the screen, but it's never reused, and pretty useless when we apply zoom on top.